src/lib/y2storage/planned_subvol.rb

```diff
@@ -127,9 +127,26 @@ def matches_arch?(target_arch = nil, &block)
       use_subvol
     end
 
+    # Create the subvolume as child of 'parent_subvol'.
+    #
+    # @param parent_subvol [::Storage::BtrfsSubvol]
+    # @param default_subvol [String] "@" or ""
+    # @param mount_prefix [String]
+    #
+    # @return [::Storage::BtrfsSubvol]
+    #
+    def create_subvol(parent_subvol, default_subvol, mount_prefix)
+      name = @path
+      name = default_subvol + "/" + path unless default_subvol.empty?
```

Comment:
Duh. I used the wrong one. But the right one gives the same result:

`irb(main):007:0> File.join("", "foo/bar")`
`=> "/foo/bar"`